documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,80 @@
+# **Metadata Forms**
+
+**Concepts**
+
+Metadata forms allow users to add structured contextual information to various entities in the Data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
+
+Metadata forms serve several key purposes:
+
+- Improve data discovery by enabling more consistent, complete, and meaningful metadata
+- Capture domain-specific or organizational metadata standards
+- Enforce governance policies around data documentation
+- Facilitate data lineage and impact analysis
+- Streamline metadata management workflows
+
+![metadata form levels](pictures/metadata_forms/mf_levels.jpg#zoom#shadow)
+
+This user guide provides instructions on how to create, manage, and use metadata forms in data.all. It covers form creation, attaching forms to entities, filling out forms, managing form visibility and permissions, and enforcing form usage. Whether you are a data producer adding metadata, a data consumer leveraging it, or an administrator standardizing on formats, this guide will help you get the most value out of metadata forms.
+
+With customizable forms that can be attached across many entities, metadata forms provide a flexible and extensible mechanism for metadata management. Use this guide to learn how to configure and apply forms to address your specific metadata use cases and data governance needs.
+
+![metadata form list](pictures/metadata_forms/mf_list.png#zoom#shadow)
+
```

Comment:
This picture with the list of metadata forms could have a paragraph attached explaining what it is. "By clicking Metadata Forms in the Discovery section of the left side pane users can open a list of their (which metadata forms do they see? the ones they own? the ones they have access to?) metadata forms" "In this list they can get the info about X,Y,Z"